docs/rules/no-interpolation-inline-snapshot.md

```diff
@@ -0,0 +1,50 @@
+# Disallow string interpolation inside inline snapshots (`no-interpolation-inline-snapshot`)
+
+Prevents the use of string interpolations within `toMatchInlineSnapshot`.
+
+## Rule Details
+
+Interpolation prevents snapshots from being updated. Instead, properties should
+be overloaded with a matcher by passing the optional `propertyMatchers` argument
+to `toMatchInlineSnapshot`.
+
+Examples of **incorrect** code for this rule:
```

Comment:
Could you chuck a in/correct example for `toThrowErrorMatchingInlineSnapshot` as well?

Reply:
added in https://github.com/jest-community/eslint-plugin-jest/pull/553/commits/a405636f7128de898187ac9dd3b54e90d65529b2